examples/example_params/gail_params.json

```diff
@@ -70,7 +94,7 @@
             "AgentName": "gail_agent"
         },
         "ExpertTrajectories": {
-            "expert_path_dir": "../com_github_gail_4_bark_large_data_store/expert_trajectories/sac/merging",
+            "expert_path_dir": "../com_github_gail_4_bark_large_data_store/expert_trajectories/interaction_dataset/DR_DEU_Merging_MT_v01_shifted/single_scenario/",
```

Comment:
Is this a file with multiple scenarios? If yes we might just add the one of the single agent we are observing!

Reply:
I tried multiple approaches:
- Train on all expert trajectories
- Train on only his WHOLE trajectory
- Train on only his trajectory cutted to the timeframe we also use in the training simulation
- Train on the six agents that are valid during the simulation [63 .. 68] (This is the current single_scenario folder) 
- Train on different subsets of 50, 100, 150, 250, 300, 400, 500, all trajectories

Reply:
None did work yet :(